docs/deploying-applications/variables/sensitive-variables.md

```diff
@@ -36,6 +36,7 @@ Here are some common pitfalls to avoid:
 - **Avoid short values:** only sensitive variables with length **greater than 3** characters will be masked. This is done to prevent false positives causing excessive obfuscation of the logs. Consider 8-30 characters depending on the requirements of your deployment.
 - **Avoid common language**: see the example below of "broke", use a password generator with high entropy [like this one](http://passwordsgenerator.net/).
 - **Avoid sequences that are interpreted by your scripting language of choice**: For example, certain escape sequences like `$^` will be misinterpreted by PowerShell potentially logging out your sensitive variable in clear-text.
+- **Avoid sequences that are part of the variable substitution syntax** {#Sensitivevariables-Avoidingcommonmistakes-SubstituionSyntax}: For example, the sequence `##{` will be replaced by `#{` by logic that's part of [referencing variables](docs/deploying-applications/variables/binding-syntax.md#Bindingsyntax-Referencingvariablesinstepdefinitions).
```

Comment:
Perhaps suggest how to escape. Maybe a link to the subsitution page (https://octopus.com/docs/reference/variable-substitution-syntax) and add some more examples.

Also FYI the script 
```
write-Host "##{NotToBeReplaced}"
write-Host "###{NotToBeReplaced}"
write-Host "####{NotToBeReplaced}"
```

Produces output of:

```
#{NotToBeReplaced}
##{NotToBeReplaced}
###{NotToBeReplaced}
```